site/_includes/layouts/chrome-100.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/base.njk" %}
+
+{% block chromescripts %}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.9.1/gsap.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.9.1/ScrollTrigger.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.9.1/ScrollToPlugin.min.js"></script>
+  <script type="module" src="{{ helpers.hashForProd('/js/firebase.js') }}"></script>
+  <script type="module" src="{{ helpers.hashForProd('/js/slider.js') }}"></script>
+{% endblock %}
+
+{% block content %}
+  {% include 'partials/chrome-100-hero-section.njk' %}
+  {% include 'partials/chrome-100-card-section.njk' %}
+  {% include 'partials/chrome-100-footer-section.njk' %}
+{% endblock %}
+
+{% block chromeanalytics %}
+  <script>
+    const chromeAnchors = document.getElementsByTagName('a');
```

Comment:
Please check comments in file [site/_includes/partials/script.js](https://github.com/GoogleChrome/developer.chrome.com/pull/2415/files#diff-909fe9fc6a9c84b779b1422b36133e08e30f11f413e9267103105dc0efad6983)

Reply:
Updated the code!